tasks/task_database.c

```diff
@@ -202,12 +181,7 @@ static int task_database_iterate_playlist(
 #else
          break;
 #endif
-      case FILE_TYPE_CUE:
-         db_state->serial[0] = '\0';
-         cue_get_serial(db_state, db, name, db_state->serial);
-         db->type = DATABASE_TYPE_SERIAL_LOOKUP;
-         break;
-      case FILE_TYPE_ISO:
+	 case FILE_TYPE_ISO:
          db_state->serial[0] = '\0';
          iso_get_serial(db_state, db, name, db_state->serial);
```

Comment:
Looks like AdvanceSCENE provides CRCs for ISOs. What is the remaining system that uses Serial checking?